pkg/network/tracer/tracer.go

```diff
@@ -143,39 +149,50 @@ func NewTracer(config *config.Config) (*Tracer, error) {
 			string(probes.HttpInFlightMap):    {Type: ebpf.Hash, MaxEntries: uint32(config.MaxTrackedConnections), EditorFlag: manager.EditMaxEntries},
 		},
 	}
-	mgrOptions.ConstantEditors, err = runOffsetGuessing(config, offsetBuf)
-	if err != nil {
-		return nil, fmt.Errorf("error guessing offsets: %s", err)
-	}
 
-	closedChannelSize := defaultClosedChannelSize
-	if config.ClosedChannelSize > 0 {
-		closedChannelSize = config.ClosedChannelSize
+	var buf bytecode.AssetReader
+	if config.EnableRuntimeCompiler {
+		buf, err = getRuntimeCompiledTracer(config)
+		if err != nil {
+			log.Warnf("error compiling network tracer, falling back to pre-compiled: %s", err)
```

Comment:
Besides logging we probably want a metric reporting how many tracers are running with runtime compilation

Reply:
Can you be more specific? Do you mean an expvar metric, or something more involved?

Reply:
IMO it should be one of these telemetry bits we embed in the protobuf messages
cc @leeavital

Reply:
Is that something we want in this PR, or a separate PR later?

Reply:
I think it's totally fine to leave to a separate PR as long as we have it before rolling out to customers